gossip/c_block_callbacks.go

```diff
@@ -262,6 +268,14 @@ func consensusCallbackBeginBlockFn(
 							}
 						}
 					}
+					// Skipped txs require original order from events
+					skippedTxs := make([]uint32, 0, len(unorderedTxs))
+					for _, txNumber := range skippedByEvm {
+						txHash := orderedTxs[txNumber].Hash()
+						skippedTxs = append(skippedTxs, txPositions[txHash].EventOffset)
+					}
+					// Txs skipped by scrambler holds on the original order from events
+					block.SkippedTxs = append(skippedTxs, skippedByScrambler...)
```

Comment:
@HerbertJordan, do you think it is a good idea to add another mechanism for skipping txs? Would not be more straightforward to let scrambler only reorder txs, with no filtering, and let only the evm to skip duplicitous nonces?

Reply:
After discussion with @thaarok and @jmpike we agreed that skipping the transactionss at this point is not necessary as the EVM will handle it. It would be diffiicult to include transactions for which we cannot derive the address into the `skippedTransactions`.
So I will remove this code.

Reply:
@thaarok that was the original idea. The scrambler should only reorder the transactions. But then it turned out that there could be errors during decoding some of those transactions, during what is now the scrambling. Those transactions should be skipped. So it evolved into this code.

To clean this up, I think the best approach is to get rid of the list of skipped transactions all-together. This could come for free while doing the block-structure change we discussed this morning. Will know more in 1-2 days.